schemas/generic-test.schema.json

```diff
@@ -5,7 +5,7 @@
   "definitions": {
```

Comment:
Can't we remove this file as well? I don't think it's actually being used in the new arch.

Reply:
Yes I'll remove.

Reply:
Done.